### DISCUSSION TITLE
Codesandbox screenshot dashboard

### DISCUSSION THREAD

**Discussion by Mannaio** [codesandbox/codesandbox-client#6383] [Q&A]
I have some codesandboxes showing the **default-sandbox-thumbnail** and others showing the thumbnail of the codesandbox right.

See the screenshot of my sandboxes list.

Why is this happening? How is possible to generate the right screenshot in my dashboard for the each codesandbox ?

Link to codesanbox not generating screenshot => https://codesandbox.io/s/confident-cdn-rkq0c

![codesandox-screenshot](https://user-images.githubusercontent.com/1617078/148641133-01b7357a-5229-408f-b82e-aab2af2ec876.png)

**Comment by andrasbacsai** [comment]
We had a problem with our screenshot service. Sorry about that. It is fixed now. [Here](https://screenshots.codesandbox.io/rkq0c/11.png) is the latest screenshot of your sandbox as an example.

**Reply by Mannaio** [reply]
Thanks for the fix!! Awesome
